### PR TITLE
Initial commit of markdown documentation

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
This will fail until the markdown-plugin is released.

https://github.com/blackbaud/skyux-builder-plugin-stache/pull/28

### CODE REVIEW

File: src/app/content/markdown/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<app-nav>
+</app-nav>
+
+<stache
+  pageTitle="Markdown"
+  layout="sidebar"
+  showBreadcrumbs="true">
+
+  <stache-page-summary>
+    <p>The markdown component allows you to format text as a using markdown formatting text using <stache-code>stache-markdown</stache-code> tags.</p>
```
Comment (Blackbaud-LindseyRix) [inline src/app/content/markdown/index.html]: This sentence could use some improvement. It doesn't read very clearly right now.

File: src/app/content/markdown/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  <div class="stache-demo-page-result">
+    <stache-markdown>
+      ## Example Heading
+
+      - List item 1
+      - List item 2
+    </stache-markdown>
+  </div>
+
```
Comment (Blackbaud-LindseyRix) [inline src/app/content/markdown/index.html]: Should we provide a x-ref link here to a helpful page that lists common markdown?

**Comment by Blackbaud-LindseyRix** [comment]
We should also consider adding a page for Markdown under API>Components if there are any technical documentation we should note about implementing/using the component for developers who may want to enhance the feature.

**Comment by Blackbaud-BrandonJones** [comment]
Totally missed this PR, but the docs for the MD component are now live. https://github.com/blackbaud/stache2-docs/pull/87

**Pull request closed without merging**